aurora/src/luks-unlock.py

```
@@ -0,0 +1,450 @@
+    }
+    key_map.update({str(i): f"KEY_{i}" for i in range(10)})
+    key_map["-"] = "KEY_MINUS"
+    key_map["_"] = "KEY_MINUS"
+    key_map[" "] = "KEY_SPACE"
+
+    for ch in passphrase:
+        key = key_map.get(ch)
+        if key is None:
+            print(f"[luks-unlock] WARNING: no key mapping for {ch!r}", file=sys.stderr)
+            continue
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)